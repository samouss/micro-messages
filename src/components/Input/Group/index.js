// @flow

import React from 'react';
import './index.css';

type Props = {
  children: React.Element<*>,
};

const Group = ({ children, ...props }: Props) => (
  <div
    {...props}
    styleName="Group"
  >
    {children}
  </div>
);

export default Group;
