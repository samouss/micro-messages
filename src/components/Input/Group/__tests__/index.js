// @flow

import React from 'react';
import { shallow } from 'enzyme';
import Group from '../index';

describe('<Group />', () => {
  const defaultProps = {
    className: 'customClassName',
  };

  it('expect to render the component correctly', () => {
    const props = {
      ...defaultProps,
    };

    const component = shallow(
      <Group
        {...props}
      >
        <span>some content</span>
      </Group>,
    );

    expect(component).toMatchSnapshot();
  });
});
