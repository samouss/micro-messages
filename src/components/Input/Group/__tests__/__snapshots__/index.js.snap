// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Group /> expect to render the component correctly 1`] = `
<div
  className="customClassName"
  styleName="Group"
>
  <span>
    some content
  </span>
</div>
`;
